src/pages/data-visualization/getting-started/vanilla.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Get started
+tabs: ['Vanilla', 'React', 'Angular', 'Vue', 'Other frameworks']
+---
+
+import DataOptionsSection from './shared-data-options-section';
+
+<PageDescription>
+
+Carbon charts help you tell accurate and convincing stories around data with beautiful and accessible visualizations.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Carbon Charts Vanilla library</AnchorLink>
+<AnchorLink>Install</AnchorLink>
+<AnchorLink>Getting started</AnchorLink>
+<AnchorLink>Development</AnchorLink>
+<AnchorLink>Troubleshooting</AnchorLink>
+
+</AnchorLinks>
+
+## Resources
+
+<Row className="resource-card-group">
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    subTitle="Carbon Charts Core"
+    href="https://github.com/carbon-design-system/carbon-charts/tree/master/packages/core">
+    <MdxIcon name="github" />
+  </ResourceCard>
+</Column>
+
+<Column colMd={4} colLg={4} noGutterSm>
+  <ResourceCard
+    subTitle="Try Carbon Charts Vanilla components with CodeSandbox."
+    href="https://codesandbox.io/s/149vrzo62l">
+    <MdxIcon name="codesandbox" />
+  </ResourceCard>
+</Column>
+
+<Column colLg={4} colMd={4} noGutterSm>
+  <ResourceCard
+    subTitle="Download WIP charting design specifications (IBM internal link)"
+    href="https://ibm.box.com/s/e4u7jaupf6pcvqd1ulxgrt1kmsf6o4bt"
+    actionIcon="download">
+    <MdxIcon name="sketch" />
+  </ResourceCard>
+</Column>
+</Row>
+
+## Carbon Charts Vanilla library
+
+The library provides front-end developers & engineers a collection of reusable charting components to build websites and user interfaces. Adopting the library enables developers to use consistent markup, styles, and behavior in prototype and production work.
+
+## Install
+
+#### Using yarn:
```

Comment:
same as before about `:`

Reply:
@connor-leech ?

Reply:
We shouldn't have colons in headers, good call @jnm2377